packages/rs-drive-abci/src/query/v0/mod.rs

```diff
@@ -898,24 +1038,32 @@ impl<C> Platform<C> {
                     .encode_to_vec()
                 } else {
                     //todo: fix this so we return optionals
-                    let identities = check_validation_result_with_data!(self
+                    let identities = self
                         .drive
                         .fetch_full_identities_by_unique_public_key_hashes(
                             public_key_hashes.as_slice(),
                             None,
                             platform_version,
-                        ));
-                    let identities = check_validation_result_with_data!(identities
+                        )?;
+
+                    if identities.is_empty() {
```

Comment:
We requesting multiple identities, we respond with an empty set if non of them found

Reply:
Fixed